tox.ini

```diff
@@ -112,8 +115,10 @@ commands =
     format: isort --quiet --atomic h tests bin
     checkformatting: black --check h tests bin
     checkformatting: isort --quiet --check-only h tests bin
-    {tests,functests}: sh bin/create-testdb
-    tests: python -m pytest --cov --cov-report= --cov-fail-under=0 --numprocesses logical --dist loadgroup {posargs:tests/unit/h/}
+    tests: sh bin/create-testdb h_tests
+    functests: sh bin/create-testdb h_functests
+    {tests,functests}: python3 -m h.scripts.init_db --delete --create
```

Comment:
This is where tox initializes the DB once before starting the parallel pytest runners